sorc/ncep_post.fd/RQSTFLD.f

```diff
@@ -24,7 +24,7 @@ module RQSTFLD_mod
       implicit none
 !
 !     increase MXFLD each time you add a new field
-      INTEGER, PARAMETER :: MXFLD=950,MXLVL=500
+      INTEGER, PARAMETER :: MXFLD=950,MXLVL=70
```

Comment:
Per previous discussion, leave this as original 70 for now if we can.  Increasing this forces community to add more zeros to *.parm files for grib1 support, not ideal.   Goal is to keep codes in sync until we finalize the build restructuring and then freeze for community release.

Reply:
We changed MXLV=500 due to gfs v16 has 127 levels.

Reply:
Ok, would you be open to us wrapping this with a compiler directive "ifdef COMMCODE" and changing it from a .f suffix to a .F suffix.  We can remove these and change back to .f if desired after the next comm release code freeze?  
The final option is that we modify out .parm files with extra zeros, and also need to modify READCNTRL.f with correct formatting for reading those files.  This is doable, just less appealing from a user standpoint.

Reply:
We may try that option.

Reply:
Is there concern about disconnecting history associated with RQSTFLD.f if we move that file to be RQSTFLD.F?  I think git can still follow the content change in the log using the --follow option with git log.  But we would have a couple months with a different file/suffix.

Reply:
Moved RQSTFLD.f and READCNTRL.f to .F files and added compiler directives to set mxlvl to 70 just for community build.